clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystem.java

```diff
@@ -63,9 +63,48 @@ public long create(TachyonURI path, long blockSize, boolean recursive) {
   }
 
   /**
-   * Convenience function for delete recursively. This is the same as calling delete(file, true).
+   * Convenience method for {@link #createEmptyFile(TachyonURI, ClientOptions)} with default
+   * client options.
    *
-   * @param file the TachyonFile to delete recursively
+   * @param path the Tachyon path of the file
+   * @return the fileId of the created file
+   * @throws InvalidPathException if the provided path is invalid
+   * @throws FileAlreadyExistException if the file being written to already exists
+   * @throws BlockInfoException if the provided block size is invalid
+   * @throws IOException if the master cannot create the file.
+   */
+  public long createEmptyFile(TachyonURI path) throws IOException, InvalidPathException,
```

Comment:
Can the implementation for this method just call the other createEmptyFile(path, options)?